care/utils/csp/config.py

```diff
@@ -13,9 +13,18 @@ class CSProvider(enum.Enum):
 
 def get_client_config():
     config = {
-        "region_name": settings.CLOUD_REGION,
-        "aws_access_key_id": settings.FILE_UPLOAD_KEY,
-        "aws_secret_access_key": settings.FILE_UPLOAD_SECRET,
+        BucketType.PATIENT.value: {
```

Comment:
Are you sure that this change does not cause other features to break?

Reply:
I've updated the code and ensured that this is working fine locally